python/cudf/cudf/_lib/groupby.pyx

```diff
@@ -169,6 +172,19 @@ cdef class GroupBy:
         result = Table(data=result_data, index=grouped_keys)
         return result
 
+    def replace_nulls(self, Column values, replace_policy policy):
+        cdef column_view val_view = values.view()
+        cdef unique_ptr[column] c_result
+
+        with nogil:
+            c_result = move(
+                self.c_obj.get()[0].replace_nulls(val_view, policy)
+            )
+
+        result = Column.from_unique_ptr(c_result)
```

Comment:
As a note, this will return integer columns in `result` where there were categorical columns in `values`. We'll have to add back categorical information in a way similar to [Frame._copy_categories](https://github.com/rapidsai/cudf/blob/branch-0.18/python/cudf/cudf/core/frame.py#L2314).